src/ORM/Query.php

```diff
@@ -1285,6 +1300,20 @@ public function insert(array $columns, array $types = [])
         return parent::insert($columns, $types);
     }
 
+    /**
+     * Returns a new Query that has automatic field aliasing disabled.
+     *
+     * @param \Cake\ORM\Table $table The table this query is starting on
+     * @return static
+     */
+    public function subquery(Table $table)
```

Comment:
How would you see this being used in window and CTE expressions?

Reply:
The goal is being able to use `ORM\Query` without the field aliasing. So, that lets me use the same style of query building as the outer query with where/innerJoinWith/etc that references associations.